Traceback (most recent call last):
  File "/rds/general/user/ak1920/home/fyp/fyp/pytorch_models/time_domain_model_simulated.py", line 129, in <module>
    train_multiple(csdy, f2, name)
TypeError: train_multiple() takes 2 positional arguments but 3 were given
